core/src/main/scala/com/rackspace/com/papi/components/checker/wadl/WADLCheckerBuilder.scala

```diff
@@ -191,17 +192,24 @@ class WADLCheckerBuilder(protected[wadl] var wadl : WADLNormalizer) extends Lazy
         val deviceHandler = getTransformerHandler(raxDeviceTemplates)
         deviceHandler.setResult(new SAXResult(buildHandler))
 
-        if(c.enableRaxRolesExtension){
+        var lastHandler = deviceHandler
```

Comment:
It's the first used handler and the last handler in the code.  I can change the name.